src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/RegisterTei/RegisterTei.component.tsx

```diff
@@ -1,17 +1,21 @@
-// @flow
-import React, { type ComponentType, useContext, useCallback } from 'react';
+import React, { useContext, useCallback } from 'react';
 import { compose } from 'redux';
-import { withStyles } from '@material-ui/core/styles';
+import { withStyles, type WithStyles, type StyleRules } from '@material-ui/core/styles';
 import i18n from '@dhis2/d2-i18n';
 import { Button } from '@dhis2/ui';
 import { RegisterTeiDataEntry } from './DataEntry/RegisterTeiDataEntry.container';
 import { RegistrationSection } from './RegistrationSection';
 import { DataEntryWidgetOutput } from '../../../../DataEntryWidgetOutput/DataEntryWidgetOutput.container';
-import { ResultsPageSizeContext } from '../../../shared-contexts';
-import type { ComponentProps } from './RegisterTei.types';
+import { ResultsPageSizeContext, type ResultsPageSizeContextType } from '../../../shared-contexts';
 import { withErrorMessageHandler } from '../../../../../HOC';
+import { ComponentProps } from './RegisterTei.types';
 
-const getStyles = () => ({
+export type StyleProps = {
+    container: React.CSSProperties;
+    leftContainer: React.CSSProperties;
+};
```

Comment:
Please remove all StyleRules and StyleProps. It does not make sense. Use the getStyles or styles object directly.